datasets/glossis/utils.py

```diff
@@ -88,14 +109,14 @@ def upload_to_gee(filename, bucket, asset, wait=True, force=False):
         )
     )
 
-    logging.info(gee_cmd)
-    result = subprocess.run(gee_cmd, shell=True, capture_output=True, text=True)
+    logger.info(gee_cmd)
+    result = subprocess.run(gee_cmd, shell=True, stdout=PIPE, universal_newlines=True)
```

Comment:
Why the change to `universal_newlines`? You're not using 3.7?

Reply:
I  was changing python3.6

Reply:
text is an alias for universal_newlines